src/components/Recovery.jsx

```diff
@@ -96,23 +115,25 @@ function Recovery({ credentials, setCredentials }) {
 
       if (response === null) {
         window.alert(
-          "Password reset failed.\n\nThe server could not be accessed.  Please try again later."
```

Comment:
Thanks for the comment. What do you mean by "established typewriting rules"? Even in Microsoft Word two spaces between sentences would be considered an error.

Reply:
`<OldManTalking backInMyDay={true}>`

The first typewriters were entirely mechanical and used hammers with raised characters on them to strike an ink ribbon in order to transfer that character to the paper.  To change the font, though, it was necessary to replace all of the hammers (if, indeed, it were even possible).

[IBM's Selectric typewriters](https://en.wikipedia.org/wiki/IBM_Selectric) featured an innovative metal ball with raised characters all over it -- when a key was pressed, the ball would be rotated to the correct character before striking the ink ribbon.  Changing fonts was as simple as changing just the metal ball.

Nevertheless, all typewriter fonts had to be mono-space fonts because typewriters had no way to know how wide an individual character was (at best, letter-spacing or pitch could be adjusted).  Consequently, all characters on a page would be in neat columns.  The convention of separating sentences with two spaces was adopted to make the ends of sentences in a paragraph more apparent to the average reader.

Nowadays, word processors can use proportional fonts so the end-of-sentence issue isn't as pronounced.  Modern style guides (like the [Chicago Manual of Style](https://www.chicagomanualofstyle.org)) have therefore dropped the two-space convention and recommend using single spaces.  HTML renderers use proportional fonts (except within `<code>` and `<pre>` blocks) by default, so they automatically collapse all white-space into a single space.

**However, text editors still keep characters in neat columns and still use mono-space fonts!**  Therefore, it makes sense to separate sentences in a text file with two spaces for readability and let the HTML renderer automatically collapse them into one.

`</OldManTalking>`

*You'd be surprised by the variety of opinions people have on this matter!  I guess that a lot of it has to do with individuals' aesthetics.*

Reply:
I can change them back but I have not seen having 2 spaces in HTML at all in my previous projects. This can barely happen that we need 2 spaces in HTML.